drfx/settings.py

```diff
@@ -382,6 +394,9 @@
         str,
     ),
     "GITHUB_URL": (GITHUB_URL, "Link to the github repository", str),
+    "MATRIX_ACCESS_TOKEN": (MATRIX_ACCESS_TOKEN, "Matrix user access token", str),
```

Comment:
I would like to keep "secrets" out of the database and just have them in the `settings_local.py` file